features/logs.js

```diff
@@ -0,0 +1,35 @@
+function logging(app,con){
+    
+    app.post('/log',function(req,res){
+        const date = new Date().toISOString().replace('T', ' ').slice(0,-1);
+        const email = req.body.email;
+        const eventname = req.body.eventname
+        con.query(`insert into logs values("${date}","${email}","${eventname}")`)
+        res.end(JSON.stringify({'message':'success'}));
+    })
+
+    app.get('/getlogs/:email',async function(req,res){
+        const email = req.params.email;
+        if (await usercheck(email)){
```

Comment:
Instead of a check email here, we can add a foreign key constraint in the database. Then just return the error message. But overall good job!!